index.html

```diff
@@ -6,276 +6,307 @@
-          I'm always <br />
-          interested in <br />
-          hearing about <br />
-          new projects, so <br />
-          if you'd like to <br />
-          chat please get <br />
-          in touch.
-        </h2>
-        <form autocomplete="off" action="https://formspree.io/f/xzbqljbl" method="post" class="input-form">
-          <input id="name-input" onchange="saveValue(this);" class="full-name" type="text" placeholder="Full name" maxlength="30" name="full-name" required />
-          <input id="email-input" onchange="saveValue(this);" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" type="email" placeholder="Email address" name="email" oninvalid="InvalidMsg(this);" oninput="InvalidMsg(this);" required />
-          <textarea id="textarea-input" onchange="saveValue(this);" class="details" id="details" cols="20" rows="30" placeholder="Hey I’d like to connect to..." maxlength="500" required name="message"></textarea>
-          <button name="submit" class="contact-btn-submit" type="submit" title="press">Get in touch</button>
-          <p class="contact-email">yesuf023@gmail.com</p>
-        </form>
-        <ul class="social-media-icons-contact">
-          <li class="twitter-icon">
-            <a href="">
-              <img src="assets/images/twitter.svg" alt="twitter" />
-            </a>
+      
+    <footer class="footer-container" id="footer-container-id">
+      <div class="footer-form">
+          <h3 class="footer-title">
+              I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h3>
+          <form action="https://formspree.io/f/xzbqljbl" method="post" class="form-input">
+              <ul class="form-fields">
+                  <li class="input-name">
+                      <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
+
+                  </li>
+                  <li class="input-full-name">
+                      <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name"
+                          required>
+                      <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name"
+                          required>
+                  </li>
+                  <li class="input-email-li">
+                      <input class="input-email" type="email" name="email" id="email" placeholder="Email address"
+                          required>
+                  </li>
+                  <li>
+                      <textarea name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                          placeholder="Enter text here" required></textarea>
+                  </li>
+                  <li class="submit-button">
+                      <button type="submit">
+                          Get in touch
+                      </button><br>
+                      <small></small>
+                  </li>
```

Comment:
![image](https://github.com/yesuf023/Portfolio/assets/60083437/1dcef5ba-2edc-4895-95a0-1cabf9535edf)

- [ ] This `button` should be aligned with the form as per the given design. Kindly fix this issue.